src/gt4py/next/otf/binding/cpp_interface.py

```diff
@@ -68,7 +70,9 @@ def render_function_declaration(function: interface.Function, body: str) -> str:
     template_params = [
         f"class BufferT{index}"
         for index, param in enumerate(function.parameters)
-        if isinstance(param.type_, ts.FieldType)
+        if isinstance(
+            param.type_, (ts.FieldType, ts.TupleType)
+        )  # TODO(havogt) add support for scalar tuples
```

Comment:
```suggestion
        )  # TODO(havogt): add support for scalar tuples
```
Just a note: This implicit coupling with `_render_function_param` is somewhat ugly.